src/main/java/org/ow2/proactive/catalog/report/TableDataBuilder.java

```diff
@@ -87,15 +95,35 @@ public void buildTableData(Set<CatalogObjectMetadata> orderedObjectsPerBucket, B
 
     }
 
+    private String getGenericInfo(CatalogObjectMetadata catalogObject) {
+        return catalogObject.getMetadataList()
+                            .stream()
+                            .filter(metadata -> metadata.getLabel().equals("generic_information"))
+                            .map(metadata -> "<li><b>" + metadata.getKey() + "</b> = " + metadata.getValue() + "</li>")
+                            .collect(Collectors.joining(""));
+
+    }
+
+    private String getVariable(CatalogObjectMetadata catalogObject) {
```

Comment:
Maybe we can rename with plural: `getVariables`, if it applies. Same comment for `getGenericInfo`.